R/tune_bayes.R

```diff
@@ -192,7 +192,8 @@ tune_bayes.model_spec <- function(object,
                                   metrics = NULL,
                                   objective = exp_improve(),
                                   initial = 5,
-                                  control = control_bayes()) {
+                                  control = control_bayes(),
+                                  eval_times = NULL) {
```

Comment:
How do we feel about `eval_times` as an argument?

Reply:
I like it, not too overly long

Reply:
On board!🙂